code/game/objects/machinery/cic_maptable.dm

```diff
@@ -31,6 +33,7 @@
 	if(!map)
 		map = SSminimaps.fetch_minimap_object(targetted_zlevel, allowed_flags)
 	user.client.screen += map
+	interactees += user
```

Comment:
harddel protection

Reply:
deletion triggers on_unset_interaction, so it shouldn't harddel